docs/architecture/UI_STACK.md

```
@@ -0,0 +1,21 @@
+# Tech Stack Decision Record
+
+## UI Architecture (Post-Reboot)
+
+### Styling: Tailwind CSS (v4)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)